src/main/java/org/jenkins/tools/test/model/MavenPom.java

```diff
@@ -83,7 +83,7 @@ public void transformPom(MavenCoordinates coreCoordinates) throws PomTransformat
         File pom = new File(rootDir.getAbsolutePath() + "/" + pomFileName);
         File backupPom = new File(rootDir.getAbsolutePath() + "/" + pomFileName + ".backup");
         try {
-            FileUtils.rename(pom, backupPom);
+            Files.move(pom.toPath(), backupPom.toPath(), StandardCopyOption.REPLACE_EXISTING);
```

Comment:
Preferring standard Java Platform functionality to Plexus FileUtils for maintainability.